packages/examples/packages/cronjobs/src/index.test.ts

```diff
@@ -22,7 +22,8 @@ describe('onCronjob', () => {
         ]),
       );
 
-      await ui.ok();
+      // TODO(ritave): Fix types in SnapInterface
+      await (ui as any).ok();
```

Comment:
Why is this happening now?

Reply:
It was always happening, but jest doesn't check types